app/views/users/show.html.erb

```diff
@@ -1,8 +1,9 @@
 <main>
-  <section class="users-section">
+  <section class="users-section d-flex justify-content-center flex-column align-items-center">
     <article class="users-info">
-      <div class="users-section-card">
-        <img src="<%= @user.photo %>" alt="Where the user face would be">
+      <div class="users-section-card d-flex">
+        <%= image_tag "user-regular.svg" %>
+
         <h4><%= @user.name %></h4>
         <p>Number of Posts: <%= @user.posts_counter %></p>
```

Comment:
- [ ] When I create a post, the posts_counter increments by 2 instead of 1. Kindly fix that.